app/src/main/java/ir/jaamebaade/jaamebaade_client/view/components/SearchBar.kt

```diff
@@ -145,5 +177,67 @@ fun SearchBar(
                 }
             }
         }
+
+        if (searchHistory != null && !isSearchIconClicked) {
```

Comment:
instead of the if condition, use `AnimateVisibility` component and use `fadeIn` and `fadeOut` animations